Expands Shoukou's medical

Adrian16199 opened the pull request:
Everything from the right of medical,
From the right of medical up to the bridge needs to be checked because I moved ALL OF IT.

<!-- Please read these guidelines before opening your PR: https://docs.spacestation14.io/en/getting-started/pr-guideline -->
<!-- The text between the arrows are comments - they will not be visible on your PR. -->

## About the PR
<!-- What did you change in this PR? -->
I have expanded medical and changed bridge a bit while I was moving the entire court, AI room and bridge to the right
## Why / Balance
<!-- Why was it changed? Link any discussions or issues here. Please discuss how this would affect game balance. -->
To give medical some love.
## Technical details
<!-- If this is a code change, summarize at high level how your new code works. This makes it easier to review. -->

## Media
<!-- 
PRs which make ingame changes (adding clothing, items, new features, etc) are required to have media attached that showcase the changes.
Small fixes/refactors are exempt.
Any media may be used in SS14 progress reports, with clear credit given.

If you're unsure whether your PR will require media, ask a maintainer.

Check the box below to confirm that you have in fact seen this (put an X in the brackets, like [X]):
-->

- [ ] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes
<!--
List any breaking changes, including namespace, public class/method/field changes, prototype renames; and provide instructions for fixing them. This will be pasted in #codebase-changes.
-->

**Changelog**
<!--
Make players aware of new features and changes that could affect how they play the game by adding a Changelog entry. Please read the Changelog guidelines located at: https://docs.spacestation14.io/en/getting-started/pr-guideline#changelog
-->

<!--
Make sure to take this Changelog template out of the comment block in order for it to show up.
:cl:
- add: Added fun!
- remove: Removed fun!
- tweak: Changed fun!
- fix: Fixed fun!
-->

Adrian16199 committed:
Expands Shoukou's medical

Everything from the right of medical,
From the right of medical up to the bridge needs to be checked because I moved ALL OF IT.

Adrian16199 committed:
Update shoukou.yml

IamVelcroboy commented:
See notes in Discord but a summary is:
- remove windoor in command bar
- name fax machines, in bridge, med, reporter
- add/link buttons to the jani signals
- add Psych sign
- Add condenser, atmos is fine
- Look at goofy sprite layer w/ centrifuge and light

Adrian16199 committed:
Update shoukou.yml

Adrian16199 commented:
Applied all suggested changes 
Plus
Adjusted the filters in cryo to be 100 instead of 1000
Made robotics have reinforced windows instead of normal windows.
Added all the missing warppoints/navigation beacons due to the massive expansion.

Adrian16199 committed:
Merge branch 'master' into Shoukou-medical-expansion
